nodes/CardDavNode/CardDavNode.node.ts

```diff
@@ -13,6 +13,7 @@ export class CardDavNode implements INodeType {
 	description: INodeTypeDescription = {
 		displayName: 'CardDAV Node',
 		name: 'cardDavNode',
+		icon: 'file:CardDavNode.svg',
```

Comment:
Should the capitalization here match the capitalization of the file name?

Reply:
Fixed